migration/v0.18.x-v0.19.x/curator-retention.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+retention=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention -e 2>&1)
+
+[ "${?}" = "1" ] && echo "No migration needed" && exit 0
+
+echo -e "Retention settings:\n$retention"
+
+otherGB=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.otherSizeGB)
+otherAge=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.otherAgeDays)
+authlogGB=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.authLogSizeGB)
+authlogAge=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.authLogAgeDays)
+kubeauditGB=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.kubeAuditSizeGB)
+kubeauditAge=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.kubAauditAgeDays)
+kubernetesGB=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.kubernetesSizeGB)
+kubernetesAge=$(yq r "${CK8S_CONFIG_PATH}"/sc-config.yaml elasticsearch.curator.retention.kubernetesAgeDays)
```

Comment:
Won't this create an issue if only some parts are set, as then they would be read as null?

Reply:
Agree on this, maybe we'll need to check if it's set in this file otherwise use from the `default/sc-config.yaml` file. Or some yq merge magic I guess

Reply:
There's no reason for us to do anything if it is not present in the `defaults` config.
But yes I will make update the script to include checks.